meinberlin/templates/account/login.html

```diff
@@ -27,17 +27,19 @@ <h1>{% translate "Login" %}</h1>
                 <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}"/>
             {% endif %}
 
-            <div class="account__password-reminder">
-                <a href="{% url 'account_reset_password' %}">{% translate "Forgot Password?" %}</a>
-            </div>
+            <div class="narrow-wrapper__cta">
+                <div class="secondary-cta">
+                    <a href="{% url 'account_reset_password' %}">{% translate "Forgot Password?" %}</a>
+                </div>
 
-            <button class="button account__button" type="submit">{% translate "Login" %}</button>
+                <button class="button button--full-width" type="submit">{% translate "Login" %}</button>
+            </div>  
```

Comment:
maybe we can drop the `narrow-wrapper__cta` and instead of `secondary-cta` call it `narrow-wrapper__secondary-cta`? That would also make it more consistent with the other html templates in this PR as the buttons there are not wrapped in `narrow-wrapper__cta`.